dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/Parent.java

```diff
@@ -50,6 +50,11 @@ public void execute(final Namespace ns, DockerfileGitHubUtil dockerfileGitHubUti
         log.info("Updating store...");
         this.dockerfileGitHubUtil.getGitHubJsonStore(ns.get(Constants.STORE)).updateStore(img, tag);
 
+        if (ns.get(Constants.SKIP_PR_CREATION)) {
```

Comment:
Can you add a utest? We need to ensure that if somebody adds other processing that is non-PR related isn't surprised.